db/migrate/20230730134633_add_birthdate_to_pets.rb

```diff
@@ -0,0 +1,5 @@
+class AddBirthdateToPets < ActiveRecord::Migration[7.0]
+  def change
+    add_column :pets, :birth_date, :datetime
```

Comment:
Could we add a not null constraint here?

Reply:
done